src/assets/App.css

```diff
@@ -0,0 +1,44 @@
+.App {
```

Comment:
- [ ] Your style looks good. However, in this milestone make sure the styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/), and all elements in the design are added. To see what is difference between your and Zeplin designs, please take a look at the screenshots below 👇 
- Yours(differences marked with red color) 👇 
![Screenshot (224)_LI](https://user-images.githubusercontent.com/67087664/156739274-c89bc165-f70b-4844-b0ec-eb31547a870f.jpg)
- Zeplin design 👇 
![Screenshot (202)](https://user-images.githubusercontent.com/67087664/156739331-c3c9d31f-3523-4407-9f8f-27697bd96196.png)

Please try your best to make your styling exacly the same as provided on Zeplin.

Reply:
I totally confused with how the code review system works. I am repeating this week and my styling was approved last week exactly the way it is but this week am rejected

Reply:
Hi @den-cloud. I am sorry for that confusion. I have reviewed your project according to the requirements and I had a Zeplin design to compare with yours. Then I saw differences which I suggested you change. Because making the styling exactly the same as provided on Zeplin was one of the main goals of this milestone.